content/en/docs/instrumentation/java/automatic/_index.md

```diff
@@ -22,24 +22,28 @@ service or app code, see [Manual instrumentation](../manual).
     preferred directory. The JAR file contains the agent and instrumentation
     libraries.
 2.  Add `-javaagent:path/to/opentelemetry-javaagent.jar` and other config to
-    your JVM's startup arguments and launch your app:
+    your JVMs startup arguments and launch your app:
```

Comment:
Perhaps it should be 
```suggestion
    your JVM startup arguments and launch your app:
```
instead? Since there's only one JVM being instrumented here

Reply:
ah thanks ... I might not have read the sentence properly.